Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/ActiveEvent.cs

```diff
@@ -61,6 +61,12 @@ public GameObject AudioEmitter
             private set;
         }
 
+        public string MessageToSend
```

Comment:
I was thinking the same thing, but wasn't sure how ActiveEvent presented itself in the editor. If it's configurable in the editor I also think that the UnityEvent is the way to go!